src/drop.test-d.ts

```diff
@@ -0,0 +1,54 @@
+import { drop } from "./drop";
+import { pipe } from "./pipe";
+
+describe("data first", () => {
+  it("works on regular inputs", () => {
+    expectTypeOf(drop([1, 2, 3, 4, 5], 2)).toEqualTypeOf<[3, 4, 5]>();
+  });
+
+  it("works on empty arrays", () => {
+    expectTypeOf(drop([], 2)).toEqualTypeOf<[]>();
+  });
+
+  it("works with negative numbers", () => {
+    expectTypeOf(drop([1, 2, 3, 4, 5], -1)).toEqualTypeOf<[5]>();
```

Comment:
This is an incorrect test! The runtime doesn't do this, it would return the array as is, e.g. [1,2,3,4,5]

Reply:
Ooh, how did I miss that
good catch, thanks!